.pre-commit-config.yaml

```
@@ -1,11 +1,11 @@
 repos:
--   repo: https://github.com/ambv/black
+  - repo: https://github.com/ambv/black
     rev: stable
     hooks:
     - id: black
       language_version: python3.6 
```

bad indentation. Note that these changes to this file are already proposed in #2651
```suggestion
      - id: black
        language_version: python3.6 
```